pinot-plugins/pinot-input-format/pinot-clp-log/src/main/java/org/apache/pinot/plugin/inputformat/clplog/CLPLogMessageDecoder.java

```diff
@@ -38,18 +38,27 @@
  * details. The implementation is based on {@link org.apache.pinot.plugin.inputformat.json.JSONMessageDecoder}.
  */
 public class CLPLogMessageDecoder implements StreamMessageDecoder<byte[]> {
+  public static final String ERROR_SAMPLING_PERIOD_CONFIG_KEY = "errorSamplingPeriod";
   private static final Logger LOGGER = LoggerFactory.getLogger(CLPLogMessageDecoder.class);
+  private static final int DEFAULT_ERROR_SAMPLING_PERIOD = 10000;
```

Comment:
can you please help elaborate on this default value. is this value hardcoded? is this value overridable by a config.